lib/cli.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Openreevoo.Cli do
+  @moduledoc """
+  The command line interface that users leave their review with
+  """
+
+  def run do
+    choose_library |> get_review |> format_review |> IO.puts
+  end
+
+  defp choose_library do
```

Comment:
It would be nicer if we could decouple the logic from the IO here, and test them separately.